atomic_scanners/scanner-rpm-verify/rpm-verify

```diff
@@ -0,0 +1,10 @@
+type: scanner
+scanner_name: rpm-verify
+image_name: registry.centos.org/nshaikh/scanner-rpm-verify
```

Comment:
- Image needs to be under `pipeline-images` app-id.
- Also, looking at [PR on container index](https://github.com/CentOS/container-pipeline-service/pull/46), it seems you've used the app-id `container-pipeline` whereas you've used `nshaikh` here in `image_name`.  Both need to be same i.e. `pipeline-images`.